src/pages/databases/connect.tsx

```diff
@@ -9,10 +9,12 @@ export async function loader({ request }) {
     throw new Error("Local name is required");
   }
   const remoteName = url.searchParams.get("remoteName");
+  const endpoint = url.searchParams.get("endpoint");
   const petnames = fireproof("petname.mappings");
   const ok = await petnames.put({
-    _id: localName,
-    remoteName,
+    _id: "db:" + remoteName,
```

Comment:
this is also wrong, we should let the db assign the id, there is no uniqueness gaurantee, as we can have the same remote have different nicknames on different clients